package.json

```diff
@@ -13,8 +13,8 @@
     "!playwright-report/"
   ],
   "scripts": {
-    "proE2ETests": "npx playwright test",
-    "fossE2ETests": "npx playwright test Odoo Senaite"
+    "e2eTests": "npx playwright test",
+    "fossE2ETests": "npx playwright test odoo-openmrs senaite-openmrs"
```

Comment:
There are no strong conventions for this but looks like best it to use either lowercase or kebab-case when naming npm scripts (see [here](https://github.com/npm/cli/blob/e39d422d69c2275ed1e3a606447a9b9d87bdca4f/package.json#L216-L231) for a sample). camelCase is also possible though, but uglyish.

Therefore... I'll suggest:
`e2e-tests-pro` and `e2e-tests-foss`

---

Question, what is `senaite-openmrs`? 🤔  I'm asking because depending on what it is I would expect rather the alphabetical order `openmrs-senaite`.

Reply:
`senaite-openmrs ` should be  `openmrs senaite`